books/system/doc/acl2-doc.lisp

```diff
@@ -52277,36 +52316,31 @@ tables in the current Hons Space."
  number to align with your system. See @(see books-certification) for an
  extended discussion on community book certification.)</p>
 
- <p>A successful regression build is a good indicator, but it may not tell the
- whole story. Be careful to avoid introducing code which may build on your
- local machine but fail in other environments. E.g., via dependence on
- environment variables or absolute pathnames, use of external tools without an
- appropriate <see topic='@(url build::cert_param)'>cert_param</see>, short
- timeouts which may fail on slower machines, etc.</p>
+ <h3>Required: Change the Base Branch of your PR to @('testing')</h3>
+
+ <p>When opening your PR to merge changes into the main ACL2 repository, set
+ the base branch to the @('testing') branch (or similar) (<i>not</i> the
+ @('master') branch).  Once your PR is accepted into the @('testing') branch,
+ it will be tested and merged automatically into the @('master') branch if all
+ tests pass.</p>
 
- <h3>Best Code Practices</h3>
+ <h3>Optional: Follow Best Practices for ACL2 Code</h3>
```

Comment:
Maybe something stronger than "Optional", like "Suggested" or "Recommended". Something to indicate that the step is encouraged, just not policed like the others.

Same for the other uses of "Optional" in headers.